tests/test_connection.py

```diff
@@ -1115,7 +1115,7 @@ def test_fundee_forget_funding_tx_unconfirmed(node_factory, bitcoind):
     time.sleep(1)
 
     def mock_sendrawtransaction(r):
-        return {'error': 'sendrawtransaction disabled'}
+        return {'42': 'sendrawtransaction disabled'}
```

Comment:
This seems wrong...

Reply:
This turns out to be a completely different problem; I've written a fix for it.

But it does make the fundchannel *fail*, which is what your patch wants to do!  So I fixed that too...

Reply:
Ah yes I see now, in that test we need to catch the exception, triggered by `fundchannel`'s fail, to complete the test. Will wait for your fix.